scripts/Update-OnETagChanged.ps1

```diff
@@ -7,7 +7,9 @@ function Update-OnETagChanged() {
     [uri]$execUrl,
     [string]$saveFile = ".\info",
     [scriptblock]$OnETagChanged,
-    [scriptblock]$OnUpdated
+    [scriptblock]$OnUpdated,
+	  [string]$kind,
```

Comment:
Please fix the indentation used in this file, all places you have used a tab intend should have 2 spaces instead of a single tab.